src/libraries/System.Data.OleDb/src/OleDbStruct.cs

```diff
@@ -488,4 +569,43 @@ internal struct tagDBPROPIDSET
         internal int cPropertyIDs;
         internal Guid guidPropertySet;
     }
+
+    internal static class OleDbStructHelpers
+    {
+        internal static ItagDBPROPINFO CreateTagDbPropInfo()
```

Comment:
Not super important, but I did like your previous idea of changing these methods to be of the form `internal static ItagDBPROPINFO CreateTagDbPropInfo() => ODB.IsRunningOnX86 ? new tagDBPROPINFO_x86() : new tagDBPROPINFO();`.  I had some more complex stuff in these methods before, but now that it's so simple, a one-liner would look cleaner.

Reply:
Done.